integration-tests/src/test/kotlin/org/stellar/anchor/platform/DatabaseMigrationTests.kt

```diff
@@ -5,33 +5,33 @@ import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
 
 val PostgresConfig =
-  TestConfig("default").also {
-    it.env["run_all_servers"] = "false"
-    it.env["run_sep_server"] = "true"
-    it.env["data.flyway_enabled"] = "true"
-  }
+    TestConfig("default").also {
```

Comment:
Did the `ktfmt` config change?

Reply:
No, everywhere else it's 2 spaces

Reply:
It is strange the ktfmt in the Gradle build does not catch it.